pom.xml

```diff
@@ -46,6 +46,17 @@
         </dependency>
     </dependencies>
 
+    <!-- lock dependencies to versions included in Jenkins -->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>11.0.1</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Bad idea. If you want to pick up the version from Jenkins, add an `<exclude>` to the dependency that would otherwise suck in a newer Guava.